client/src/templates/Challenges/classic/classic.css

```diff
@@ -17,53 +17,50 @@
   display: none !important;
 }
 
-.action-row {
+.monaco-editor-tabs {
   padding: 10px;
   border-bottom: 1px solid var(--quaternary-background);
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(8rem, 1fr));
 }
 
-.monaco-editor-tabs [aria-expanded='true'],
-.action-row [aria-expanded='true'] {
+.monaco-editor-tabs [aria-expanded='true'] {
   border-color: var(--secondary-color);
   background-color: var(--secondary-color);
   color: var(--secondary-background);
 }
 
-.tabs-row {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-}
-
-.tabs-row button,
 .monaco-editor-tabs button {
   padding: 6px 12px;
   border-width: 2px;
   border-style: solid;
 }
 
-.monaco-editor-tabs {
-  display: flex;
-  margin-inline-end: auto;
+.preview-buttons-section {
+  display: inline-flex;
+  grid-column-end: -1;
 }
 
-.monaco-editor-tabs button:not(:first-child) {
-  border-left: none;
-}
-
-.panel-display-tabs button:first-child,
-.tabs-row > button:first-child {
-  margin: 0 10px 0 0;
+.editor-console-button {
+  width: 6rem;
+  margin-inline-start: 1.5rem;
+  grid-column-end: -2;
 }
```

Comment:
This is the part, that will makes the Console button, look closer to the Preview button, than how it looks in the production.

Didn't want to position it perfectly and making it pixel perfect, because I think there will be better solution for this later, and it takes the same effort of finishing the 95% of the issue to finish the last 5% and position it pixel perfect.

You are more than welcome to mess with, and create the perfect margins and width.